docker/rholang-cli/rhoscala

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+input=$(realpath $1)
```

Comment:
```
        ^-- SC2046: Quote this to prevent word splitting.
                   ^-- SC2086: Double quote to prevent globbing and word splitting.
```

says shellcheck

Reply:
This is wrong. It's correct as written.

Reply:
It's suggesting `input=$(realpath "$1")`

Otherwise...
```
$ x='/path/to/My Favorite File'
$ input=$(realpath $x)
realpath: /path/to/My: No such file or directory
```